search-enterprise/latest/admin/index.md

```diff
@@ -371,6 +371,48 @@ select to_char(to_timestamp(min(commit_time_ms)/1000),'YYYYMMDDHHMI') as fromTim
 
 Because it is easier to have **unbalanced partitions** when indexing by date range, it is recommended to use this strategy only when we are interested in indexing a specific date range, while it is recommended to perform a reindex by id range when we need to perform a **full reindex**.
 
+### Troubleshooting
+
+#### Search for failed Transformations
+
+You can search for all documents that have failed to transform by running a simple query using Kibana or the Elasticsearch REST API:
+
+```json
+{
+  "query": {
+      "term": {
+        "cm%3Acontent%2Etr_status": {
+          "value": "TRANSFORM_FAILED"
+        }
+      }
+  }
+}
+```
+
+#### Debug queries generated for Elasticsearch
+
+Enabling slowly queries in Elasticsearch with threshold 0 seconds, should dump every query received by Elasticsearch in the logs.
+
+```bash
+$ curl -XPUT "http://elasticsearch:9200/alfresco/_settings" -H 'Content-Type: application/json' -d'{  
+ "index.search.slowlog.threshold.query.warn": "0s",  
+ "index.search.slowlog.threshold.query.info": "0s",
+ "index.search.slowlog.threshold.query.debug": "0s",  
+ "index.search.slowlog.threshold.query.trace": "0s",  
+ "index.search.slowlog.threshold.fetch.warn": "0s",  
+ "index.search.slowlog.threshold.fetch.info": "0s",  
+ "index.search.slowlog.threshold.fetch.debug": "0s",  
+ "index.search.slowlog.threshold.fetch.trace": "0s",  
+ "index.indexing.slowlog.threshold.index.warn": "0s",  
+ "index.indexing.slowlog.threshold.index.info": "0s",  
+ "index.indexing.slowlog.threshold.index.debug": "0s",  
+ "index.indexing.slowlog.threshold.index.trace": "0s", 
+ "index.indexing.slowlog.level": "trace",  
+ "index.indexing.slowlog.source": "1000"}'
+```
+
+> **Note:** This setting is only recommended when debugging.
```

Comment:
This note ends up with H3 styling

Reply:
Not a part of this review. Covered in another PR.